Ryujinx/Ui/MainWindow.cs

```diff
@@ -306,6 +308,14 @@ private void UpdateDockedModeState(object sender, ReactiveEventArgs<bool> e)
             }
         }
 
+        private void UpdateAudioVolumeState(object sender, ReactiveEventArgs<float> e)
+        {
+            if(_emulationContext != null)
```

Comment:
```suggestion
            if (_emulationContext != null)
```
It could also be simplified to just:
```cs
_emulationContext?.SetVolume(e.NewValue);
```

Reply:
If only something as beautiful as this existed in the other languages I work with :)